webpack.config.js

```diff
@@ -13,14 +13,9 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.m?js$/,
+        test: /\.(js|jsx)$/,
         exclude: /node_modules/,
-        use: {
-          loader: 'babel-loader',
-          options: {
-            presets: ['@babel/preset-env'],
-          },
-        },
+        use: 'babel-loader',
       },
       {
         test: /\.css$/i,
```

Comment:
set the use property to this
```
[
          'style-loader',
          {
            loader: 'css-loader',
            options: {
              modules: true,
            },
          },
        ]
```

That should enable css modules. We can discuss it further at the next standup meeting.

Reply:
It is already listed below this line. If it needs to be set up differently, let's chat about it during our standup tomorrow.

Reply:
Just seeing this PR. Assume its been resolved from this morning.